docs/_snippets/storybook-testing-addon-set-annotations.md

```diff
@@ -42,17 +45,18 @@ export default mergeConfig(
       clearMocks: true,
       setupFiles: './src/setupTests.ts', //👈 Our configuration file enabled here
     },
-  }),
+  })
 );
 ```
 
 ```js filename="setupFile.js|ts" renderer="vue" language="js"
 // Storybook's preview file location
-import * as globalStorybookConfig from './.storybook/preview';
+import * as previewAnnotations from './.storybook/preview';
 
 import { setProjectAnnotations } from '@storybook/vue3';
+import { render as testingLibraryRender } from '@testing-library/vue';
 
-setProjectAnnotations(globalStorybookConfig);
+setProjectAnnotations([previewAnnotations, { testingLibraryRender }]);
```

Comment:
I'm ok with leaving this as is but we have many formats of setProjectAnnotations throughout the docs:

```ts
setProjectAnnotations([
  previewAnnotations, 
  addonAnnotations, 
  // you MUST ...
  { testingLibraryRender }
]);
```

```ts
setProjectAnnotations([previewAnnotations, { testingLibraryRender }]);
```

```ts
setProjectAnnotations(previewAnnotations);
```

```ts
const annotations = setProjectAnnotations([previewAnnotations, { testingLibraryRender }]);
beforeAll(annotations.beforeAll)
```